bitcoin/src/taproot.rs

```diff
@@ -10,15 +10,17 @@ use core::convert::TryFrom;
 use core::fmt;
 
 use bitcoin_internals::write_err;
-use secp256k1::{self, Scalar, Secp256k1};
+use secp256k1::{self, Scalar};
 
 use crate::consensus::Encodable;
-use crate::crypto::key::{TapTweak, TweakedPublicKey, UntweakedPublicKey, XOnlyPublicKey};
 use crate::hashes::{sha256t_hash_newtype, Hash, HashEngine};
 use crate::prelude::*;
 use crate::{io, Script, ScriptBuf};
 
+pub use secp256k1::{constants, Secp256k1, KeyPair, XOnlyPublicKey, Verification, Parity};
```

Comment:
I'm not really sure about these. It makes it look like these are taproot versions but they are not. The only items loosely related to taproot are `XOnlyPublicKey` and `Parity`.

Reply:
Yup, agreed. I think `KeyPair`  should also be included. `XOnlyPublicKey`, `KeyPair` and `Parity` can be exposed from secp256k1